tfhe/src/shortint/server_key/mod.rs

```diff
@@ -22,6 +22,7 @@ pub use bivariate_pbs::{
     BivariateLookupTableMutView, BivariateLookupTableOwned, BivariateLookupTableView,
 };
 pub use compressed::{CompressedServerKey, ShortintCompressedBootstrappingKey};
+pub(crate) use scalar_mul::unchecked_scalar_mul_assign;
```

Comment:
what for ?

Reply:
To do a multiplication before the packing keyswitch without needing a server key